config/src/config/consensus_config.rs

```diff
@@ -91,6 +99,11 @@ impl Default for ConsensusConfig {
             intra_consensus_channel_buffer_size: 10,
             quorum_store_configs: QuorumStoreConfig::default(),
 
+            consensus_backpressure: ConsensusBackpressureValues {
```

Comment:
I'd probably name it backpressure_configs, since we also use the consensus_backpressure as bool to indicate whether it's on or off